build.gradle

```diff
@@ -48,6 +48,8 @@ dependencies {
 	testCompile 'junit:junit:4.12'
 	compile 'org.apache.commons:commons-lang3:3.2.1'
 	compile 'commons-net:commons-net:3.3'
+	compile 'net.java.dev.jna:jna:4.2.2'
```

Comment:
To read port list from registry I used JNA library. Seems to me a must-have dependency for a library interacting with hardware resources..